### ISSUE TITLE
Modification Request  - LoveFilm Aspect widget

### ISSUE THREAD

**Issue by SerpentDrago** [jurialmunkey/skin.arctic.horizon#298]
Focused 
![image](https://user-images.githubusercontent.com/21133858/96047823-aa700d80-0e43-11eb-9dcf-5f3ed29c7bbc.png)
Unfocused 
![image](https://user-images.githubusercontent.com/21133858/96047989-f9b63e00-0e43-11eb-8ad5-7f7961e3a8b3.png)

Currently the Skins primary highlight color completely overlays / highlights this widget . IMHO it looks bad , and would be nicer if it was just a border , like what happens for most any other widget . 

just a suggestion , thanks ! LOVE the widget by the way :)

**Comment by drinfernoo** [comment]
To be fair, this is also the behavior for landscape widgets with text panels on. I do agree, though, that the text is a bit hard to read, especially with lighter colors.

**Comment by alKODIque** [comment]
For "Love Film" view, I also note that when I select (in additional rating) "Prefer numeric star rating" , there is no rating at all showing in the widget.

**Comment by SerpentDrago** [comment]
Sounds like Love film , needs some love :)

**Comment by jurialmunkey** [comment]
> 
> 
> To be fair, this is also the behavior for landscape widgets with text panels on. I do agree, though, that the text is a bit hard to read, especially with lighter colors.

Yeah it's mostly just because it's a standardized include. I'll try a few things out when I get the chance!

**Comment by unkn0wn234** [comment]
+1 on this
